recipes/nodejs/all/conanfile.py

```diff
@@ -1,8 +1,9 @@
 import os
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.errors import ConanInvalidConfiguration
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.53.0"
```

Comment:
Get and copy were very early so we can drop this :)

```suggestion
required_conan_version = ">=1.47.0"
```

Reply:
Done